crates/kv-store/src/sstable.rs

```diff
@@ -0,0 +1,1026 @@
+use std::{fmt::Debug, ops::Bound, sync::Arc};
+
+use bytes::{Buf, BufMut, Bytes};
+use fxhash::FxHashSet;
+use loro_common::{LoroError, LoroResult};
+use once_cell::sync::OnceCell;
+
+use super::block::BlockIter;
+use crate::{
+    block::{Block, BlockBuilder},
+    compress::CompressionType,
+    iter::KvIterator,
+    utils::{get_u16_le, get_u32_le, get_u8_le},
+};
+
+pub(crate) const XXH_SEED: u32 = u32::from_le_bytes(*b"LORO");
+const MAGIC_NUMBER: [u8; 4] = *b"LORO";
```

Comment:
Shuold be called MAGIC_BYTES. It has different meaning than magic number. Both are established terms.